dts/arm/microchip/mec172xnsz.dtsi

```diff
@@ -688,13 +688,14 @@
 			status = "disabled";
 		};
 		adc0: adc@40007c00 {
-			compatible = "microchip,xec-adc-v2";
+			compatible = "microchip,xec-adc";
 			reg = <0x40007c00 0x90>;
 			interrupts = <78 0>, <79 0>;
 			girqs = <17 8>, <17 9>;
 			pcrs = <3 3>;
 			status = "disabled";
 			#io-channel-cells = <1>;
+			clktime = <1>;
```

Comment:
@Manimaran-A same here with clktime default value

Reply:
Ok, i update the configuration by 32